app/src/main/java/com/heapixLearn/discovery/domain/contact/DumbServerContactManager.java

```diff
@@ -0,0 +1,35 @@
+package com.heapixLearn.discovery.domain.contact;
+
+import com.heapixLearn.discovery.logic.contact.LogicContact;
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.ServerStoreContactManager;
+
+import java.util.ArrayList;
+import java.util.concurrent.FutureTask;
+
+public class DumbServerContactManager implements ServerStoreContactManager {
```

Comment:
@dianaprimshits unused class should be removed

Reply:
@YodaMike Fixed